src/app/public/testing/infinite-scroll/infinite-scroll-fixture.ts

```diff
@@ -22,20 +28,20 @@ export class SkyInfiniteScrollFixture {
     this.waitForComponentInitialization();
   }
 
-  public isEnabled(): boolean {
+  public get infiniteScrollIsEnabled(): boolean {
```

Comment:
Do we really need this property? If you look at the infinite-scroll component, `enabled` exists only to determine the visibility of the button. Since we already have a `loadMoreButtonIsVisible` property, I think we're covered. You'd never have a case where one of these properties is out of sync with the other. 
Also, I can't think of many times where we are getting at the component instance of an internal component in our fixtures. Plus, `debugElement.componentInstance` being an `any` makes me weary of the brittleness of this getter. I'd vote for removing it all together - what do you think?